bot/__init__.py

```diff
@@ -168,6 +171,46 @@ def _get_word(self) -> str:
         result = self.buffer[self.previous:self.index + (pos+2)]
         self.index += 2
 
+    # Check if a command in the form of `bot.tags['ask'] {= 'whatever'}` was used
```

Comment:
What are these curly brackets for?

Reply:
supposed to stand for "optional thing", since both `bot.tags['ask']` and `bot.tags['ask'] = 'whatever'` would work. I would've used `[...]`, but since that was already used there, I thought that was a bit unclear

Reply:
gotcha

Reply:
I'm not a fan, it's not obvious that they signify optional input. Just use more words to say it in a more obvious way please.